### PR TITLE
docs: clarify community feature request voting

### PR DESCRIPTION
**Pull request by markgoho**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [x] Other... Please describe:
A small update to the community feature request voting explanation

## What is the current behavior?
The current process by which community members vote on feature requests is described, but is missing some important details like where to cast the vote and what votes are counted. Additionally the length of voting is wrong (60 days). The bot allows for 20 days of voting.

Issue Number: N/A

## What is the new behavior?
Updated language, fixed voting length.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by josephperrott — APPROVED** [review]
LGTM

Review-for: dev-infra

**Comment by josephperrott** [comment]
@mgechev Can we have your review on this PR as well?

File: docs/GITHUB_PROCESS.md [hunk 1]
```diff
@@ -28,16 +28,17 @@ initiates a voting process.
 
 ## Voting phase
 
-To include the community in the feature request process, we open voting for 60 days. Anyone can cast
-a vote for the request with a thumbs-up (👍) reaction. When a feature request reaches 20 or more
-upvotes, we formally consider the feature request. Alternatively, the bot closes the request.
+To include the community in the feature request process, we open voting for 20 days. Anyone can cast
```
Comment (mgechev) [inline docs/GITHUB_PROCESS.md:31]: That's only for issues older than 60 days.

**Review by mgechev — CHANGES_REQUESTED** [review]
I'd suggest updating to only reflect the 20 day rule for old feature requests. I'll have a look at the logic for new feature requests today and once this works as expected, we can add more details.

**Comment by mgechev** [comment]
I had a look today and it seems like the bot is working as expected.

The way the process works:
- New feature requests get 60 days period to collect 20 or more 👍 to be added for consideration
- Old feature requests (created 60+ days ago), get 20 days to collect 20 or more 👍

In the meantime, anyone who can add labels to issues and is familiar with our roadmap can add `in backlog` label to pause the voting process in case they decide the request is already part of an existing prioritized effort.

**Comment by markgoho** [comment]
@mgechev updated the docs change to reflect your investigation

**Review by mgechev — APPROVED** [review]

**Pull request closed without merging**